assets/locales/en/main.json

```diff
@@ -59,9 +59,44 @@
     "having-trouble": "Having trouble…",
     "no-street": "No street selected.",
     "trouble-loading": "We’re having trouble loading Streetmix.",
+    "error-code": "(Error {{code}}.)",
     "slow-connection-1": "Streetmix wasn’t able to connect to the Internet in awhile now.",
     "slow-connection-2": "You might want to reload the page and try again. Please note you might lose the latest change to the street. Sorry!",
     "no-connection": "Streetmix is having trouble connecting to the Internet."
+  },
+	"errors": {
+		"buttons": {
+			"home-button": "Go to the homepage",
+			"sign-in-button": "Sign in again",
+			"reload-button": "Reload the page",
+			"try-again-button": "Try again",
+			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+		},
+		"page-not-found-title": "Page not found.",
+		"page-not-found-description": "Oh, boy. There is no page with this address!<br />{homeButton}",
```

Comment:
Let's remove everything after, and including, the `<br />`. We can handle that part in React, we don't need to do this in the translation string.

Reply:
Ok great, I only included it since I don't understand how the string translations get connected to the actual UI.

Reply:
No worries. This is a normal process of trying something and thinking about whether it works or not.